pkg/cli/client/server_info_cmd.go

```diff
@@ -0,0 +1,116 @@
+//go:build search
+// +build search
+
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	zerr "zotregistry.io/zot/errors"
+	"zotregistry.io/zot/pkg/api/constants"
+)
+
+func NewServerInfoCommand() *cobra.Command {
+	serverInfoCmd := &cobra.Command{
+		Use:   "server-info",
```

Comment:
Should we break this into subcmds instead?

zli registry info : return this info?
zli registry status : do a /v2 lookup?

Reply:
What would be the zli output for the /v2 lookup?

Reply:
`zli status <remote>`
does both /v2 lookup and returns server version

Reply:
@laurentiuNiculae can you post an example output for this?